learn/wavemaker-release-notes/v10-13-2.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "WaveMaker 10.13.2 - Release date: 11 April 2022"
+id: ""
+sidebar_label: "Release v10.13.2"
+---
+*WaveMaker released another version update with new features and bug fixes.*
+
+## Improvements
+---
+
+- FileUpload widget: **Clear** and **Delete** methods are introduced to clear and delete uploaded files. 
+- Introduced a new property to show characters limits for the Text widget. 
```

Comment:
Introduced a new property "Help text (character count)" on the widget to help text regarding character limit for the widget. This is useful when a developers want to show how many characters have been filled (or left) out of the maximum characters allowed.

We should also add this property in the widget documentation, https://docs.wavemaker.com/learn/app-development/widgets/form-widgets/textarea/.